scripts/increment-cargo-version.sh

```diff
@@ -108,6 +109,15 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   done
 done
 
+# Update all the documentation references
+for Doc_update in "${Doc_updates[@]}"; do
+  # Set new crate version
+  (
+    set -x
+    sed -i "$Doc_update" -e "s/v[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]/$newVersion/g"
```

Comment:
We only want to replace `currentVersion` with `newVersion`, not all occurrences of x.y.z